portal/models/procedure_codes.py

```diff
@@ -147,6 +147,38 @@ def FocalTherapy(self):
             codings=[ichom], text='Focal therapy').add_if_not_found(True)
 
     @lazyprop
+    def AndrogenDeprivationTherapySurgicalOrchiectomy(self):
+        tnth = Coding(
+            system=TRUENTH_CLINICAL_CODE_SYSTEM, code='androgen deprivation therapy - surgical orchiectomy',
+            display='None').add_if_not_found(True)
```

Comment:
I believe the `display` field should be populated.  Perhaps the list in the config files could be used?